bundles/framework/timeseries/view/TimeseriesControlPlugin.js

```diff
@@ -0,0 +1,642 @@
+        this._uiState.rangeStart = range[0];
+        this._uiState.rangeEnd = range[1];
+        this._uiState.currentTime = delegate.getCurrentTime();
+
+        me._isMobileVisible = false;
+        me._inMobileMode = false;
+
+        me._mobileDefs = {
+            buttons: {
+                'mobile-timeseries': {
+                    iconCls: 'mobile-timeseries',
+                    tooltip: '',
+                    sticky: true,
+                    toggleChangeIcon: true,
+                    show: true,
+                    callback: function () {
+                        if (me._isMobileVisible) {
+                            me.teardownUI();
+                        } else {
+                            me._isMobileVisible = true;
+                            me._buildUI(true);
+                            me.getSandbox().getService('Oskari.userinterface.component.PopupService').closeAllPopups(false);
+                        }
+                    }
+                }
+            },
+            buttonGroup: 'mobile-toolbar'
+        };
+    }, {
+        __fullAxisYPos: 35,
+        __localeData: {
+            fi: {
+                "dateTime": "%A, %-d. %Bta %Y klo %X",
+                "date": "%-d.%-m.%Y",
+                "time": "%H:%M:%S",
+                "periods": ["a.m.", "p.m."],
+                "days": ["sunnuntai", "maanantai", "tiistai", "keskiviikko", "torstai", "perjantai", "lauantai"],
+                "shortDays": ["Su", "Ma", "Ti", "Ke", "To", "Pe", "La"],
+                "months": ["tammikuu", "helmikuu", "maaliskuu", "huhtikuu", "toukokuu", "kesäkuu", "heinäkuu", "elokuu", "syyskuu", "lokakuu", "marraskuu", "joulukuu"],
+                "shortMonths": ["Tammi", "Helmi", "Maalis", "Huhti", "Touko", "Kesä", "Heinä", "Elo", "Syys", "Loka", "Marras", "Joulu"]
+            },
+            sv: {
+                "dateTime": "%A den %d %B %Y %X",
+                "date": "%Y-%m-%d",
+                "time": "%H:%M:%S",
+                "periods": ["fm", "em"],
+                "days": ["Söndag", "Måndag", "Tisdag", "Onsdag", "Torsdag", "Fredag", "Lördag"],
+                "shortDays": ["Sön", "Mån", "Tis", "Ons", "Tor", "Fre", "Lör"],
+                "months": ["Januari", "Februari", "Mars", "April", "Maj", "Juni", "Juli", "Augusti", "September", "Oktober", "November", "December"],
+                "shortMonths": ["Jan", "Feb", "Mar", "Apr", "Maj", "Jun", "Jul", "Aug", "Sep", "Okt", "Nov", "Dec"]
+            }
+        },
+
+        // Returns a function, that, when invoked, will only be triggered at most once
+        // during a given window of time. Normally, the throttled function will run
+        // as much as it can, without ever going more than once per `wait` duration;
+        // but if you'd like to disable the execution on the leading edge, pass
+        // `{leading: false}`. To disable execution on the trailing edge, ditto.
+        _throttle: function (func, wait, options) {
```

Comment:
Feels like throttle should be a function in the Oskari(.util?)-global instead of implemented in a bundle.